core/src/main/scala/com/lightbend/coursegentools/Helpers.scala

```diff
@@ -149,8 +149,10 @@ object Helpers {
         new File(projectFolder, "README.md")
     }
 
+    val commonProject = if (config.commonProjectEnabled) "common" else ""
+
     // Check all required README files are present
-    for { project <- "common" +: exercises} {
+    for { project <- commonProject +: exercises} {
```

Comment:
Isn't this a bit of a hack in the sense that, in the case that `config.commonProjectEnabled` is `false`, it will test the presence of the base folder?

Reply:
yes, if the setting is false, it will check the base folder. 
And I just found the error message is not printing the path based on the config. I'll update this PR.

Reply:
I would instead create a modified `exercises` value that has conditionally added a "common" element to it.
The addition of an empty string is a burden on anyone who's reading the code for the first time.
so something like:
```
val exercises_ammended = if (config.commonProjectEnabled) "common" +: exercises else exercises
...
for { project <- exercises_ammended) {
...
```

Reply:
I've updated the code in the last commit in a similar fashion